docs/getting-started/summaryofchanges.md

```diff
@@ -41,8 +41,28 @@ The following features and enhancements were added.
 
 #### API Mediation Layer
 
+#### ZSS
+
+- Added zssServer.sh, configure.sh, and start.sh in packaging [#177](https://github.com/zowe/zss/pull/177)
```

Comment:
Better: "ZSS is now follows the Zowe Component scheme, as part of the DESKTOP component group"